conf/nginx/st2.conf

```diff
@@ -135,5 +143,6 @@ server {
     sendfile on;
     tcp_nopush on;
     tcp_nodelay on;
+    server_tokens off;
```

Comment:
I think `server_tokens off;` would be sufficient for the `server` block and not needed to repeat for every possible `location`.
Can you try that?